src/popup/pages/SignTransaction.vue

```diff
@@ -31,7 +32,10 @@ export default defineComponent({
         }
 
         const signedTransaction = await sdk.signTransaction(
-          transaction, { networkId, app: callbackOrigin.value },
+            transaction as Encoded.Transaction, {
```

Comment:
If the first argument is placed in a separate line and the second needs to take more than one line please put all the arguments in separate lines:
```
someFunc(
  arg1,
  {
    ...
  },
);
```